Source: written by John Paul Adrian Glaubitz for Debian
Upstream: https://sourceforge.net/p/ufraw/bugs/417
Reason: fixes a compilation failure on arm architectures
--- dcraw.cc
+++ dcraw.cc
@@ -2285,7 +2285,7 @@ void CLASS quicktake_100_load_raw()
 
 void CLASS kodak_radc_load_raw()
 {
-  static const char src[] = {
+  static const signed char src[] = {
     1,1, 2,3, 3,4, 4,2, 5,7, 6,5, 7,6, 7,8,
     1,0, 2,1, 3,3, 4,4, 5,2, 6,7, 7,6, 8,5, 8,8,
     2,1, 2,3, 3,0, 3,2, 3,4, 4,6, 5,5, 6,7, 6,8,
